OpenRA.Game/Manifest.cs

```diff
@@ -45,12 +45,17 @@ public SpriteSequenceFormat(MiniYaml yaml)
 
 	public class ModMetadata
 	{
+		[TranslationReference]
 		public string Title;
 		public string Version;
 		public string Website;
 		public string WebIcon32;
+		[TranslationReference]
 		public string WindowTitle;
 		public bool Hidden;
+
+		public string TitleTranslated => TranslationProvider.GetString(Title);
```

Comment:
Do we need to expose the untranslated versions at all?

Reply:
No, untranslated versions are not referenced anywhere. We should cache these as well

Reply:
Made the original fields private.

The new fields are called only a handful of times, caching them is not required.